### DISCUSSION TITLE
Add "order by" for relational interfaces

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#15075] [Feature Requests]
Currently the relational interfaces allow us to add a "sort" value. That's fine for a lot of cases but sometimes we just want the fields to be ordered by a specific field other that the ID. 

A custom option "order by" within the relational interface would solve this 😍

**Comment by azrikahar** [comment]
Although this is not a direct built-in "order by" option as suggested, but #12820 added table view for O2M and M2M which should allow sorting similar to how headers in table layout currently does. So it should be helpful for this use case in the meantime 👍

**Reply by Dominic-Marcelino** [reply]
Thank's, that indeed would be perfect for my specific use-case.

Running on 9.16.1 but only have the "one to many" and "treeview" interfaces for a o2m relation. Based on the release notes it's included since 9.16.0. Any idea why it could be missing?

**Reply by azrikahar** [reply]
It is built into the "One to Many" (or Many to Many) Interface options, not as a "standalone" separate interface like treeview. 🙂

You can select the layout as `Table` or `List` (it used to be list only).

![chrome_2LKtlQ7xgH](https://user-images.githubusercontent.com/42867097/185891174-734a7d98-cd90-4a7c-92ee-b9f1f2862b9a.png)

**Reply by Dominic-Marcelino** [reply]
Thanks! ☺️ 
In the initial gif it was a standalone interface. Haven't noticed that it has changed later 😅

**Reply by azrikahar** [reply]
Oh great point! I've looked into the PR way after the later stages so I didn't even realize it was standalone 😄

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
